src/Compilers/CSharp/Portable/Symbols/Source/LambdaParameterSymbol.cs

```diff
@@ -22,7 +23,7 @@ internal sealed class LambdaParameterSymbol : SourceComplexParameterSymbolBase
            string name,
            bool isDiscard,
            ImmutableArray<Location> locations)
-           : base(owner, ordinal, parameterType, refKind, name, locations, syntaxRef: null, isParams: false, isExtensionMethodThis: false, scope)
+           : base(owner, ordinal, parameterType, refKind, name, locations, syntaxRef, isParams: false, isExtensionMethodThis: false, scope)
```

Comment:
`HasDefaultArgumentSyntax` below should probably be updated. We should look through the users of that API and make sure they're doing the right thing.